ddtrace/settings/config.py

```diff
@@ -31,9 +32,8 @@ def __init__(self):
         )
 
         self.env = get_env("env")
-
+        self.service = get_env("service")
```

Comment:
we could do `get_env("service", default=get_env("service", "name"))` here to just support `DD_SERVICE` and `DD_SERVICE_NAME` everywhere ?

Reply:
I added a helper function `get_service` to do this so that we could also provide a deprecation warning if ``*_SERVICE_NAME` are used.